src/features/people/PeoplePage/styled.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+export const PersonLink = styled(Link)`
+    text-decoration: none;
+    display:flex;
+    justify-content: center;
+    align-items: flex-start;
+`;
```

Comment:
Skoro ten komponent jest identyczny jak przy movies to może warto zrobić jeden wspólny w common?

Reply:
Jasna sprawa @katkowa  już poprawiłem i zrobiłem z tego jeden komponent FeatureLink ;) Jeśli chcesz możesz zerknąć, jutro rano zrobię merge'a ;)